setup.py

```diff
@@ -1,30 +1,50 @@
 from setuptools import setup, find_packages
 from os import path
 
-pkg_name = next(p for p in find_packages() if "." not in p)
+pkg_name = "element_deeplabcut"
 here = path.abspath(path.dirname(__file__))
 
 with open(path.join(here, "README.md"), "r") as f:
     long_description = f.read()
 
-with open(path.join(here, "requirements.txt")) as f:
-    requirements = f.read().splitlines()
-
 with open(path.join(here, pkg_name, "version.py")) as f:
     exec(f.read())
 
 setup(
     name=pkg_name.replace("_", "-"),
     version=__version__,
-    description="DataJoint Element for Continuous Behavior Tracking via DeepLabCut",
+    description="DeepLabCut DataJoint Element",
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="DataJoint",
-    author_email="info@vathes.com",
+    author_email="info@datajoint.com",
     license="MIT",
     url=f'https://github.com/datajoint/{pkg_name.replace("_", "-")}',
-    keywords="neuroscience behavior deeplabcut datajoint",
+    keywords="neuroscience behavior pose-estimation science datajoint",
```

Comment:
```suggestion
    keywords="neuroscience behavior deeplabcut pose-estimation science datajoint",
```
Let's leave deeplabcut in there